test/AutoDiff/derived_differentiable_properties.swift

```diff
@@ -1,12 +1,27 @@
-// RUN: %target-swift-frontend -emit-silgen %s | %FileCheck %s
+// RUN: %target-swift-frontend -emit-silgen %s | %FileCheck %s --check-prefix=CHECK-AST
```

Comment:
Is the first RUN line meant to perform `-print-ast`?

Reply:
Meant to test the AST printed in SIL.